boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuapp_0.3.0.overlay

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+
+/ {
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio2 9 GPIO_ACTIVE_HIGH>;
+			label = "Green LED 0";
+		};
```

Comment:
Why not apply only the actual differences between the revisions. i.e. just the GPIO assignments:
```
&led0 {
	gpios = <&gpio2 9 GPIO_ACTIVE_HIGH>;
};
```
and so on?

Reply:
Thank you. Removed spare lines